data/commodities.txt

```diff
@@ -709,30 +836,43 @@ trade
 		"volatile gases"
 		"waste"
 	commodity "Construction"
+		"bolts"
 		"box girders"
 		"carbon steel"
+		"cement"
 		"composite beams"
 		"composite fibers"
 		"concrete"
+		"drills"
 		"duct tape"
 		"fiberglass"
 		"girders"
+		"gravel"
 		"I-beams"
+		"jackhammers"
 		"joint plates"
 		"lumber"
+		"metal"
```

Comment:
I'm not sure `metal` really works well here because it's also a commodity category and so could get confusing if it's under the construction commodity as well.